.github/workflows/ci.yml

```diff
@@ -3,10 +3,102 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: ${{ github.event_name == 'pull_request' }}
 on:
-  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+env:
+  COMPOSE_PROJECT_NAME: ${{ github.ref_name }}
+  DOCKER_REGISTRY_USR: ${{ secrets.GHA_DOCKERHUB_PULL_USER }}
+  DOCKER_REGISTRY_PSW: ${{ secrets.GHA_KONG_ORG_DOCKERHUB_PUBLIC_TOKEN }}
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - run: echo "Hello, World!"
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: checkout kong portal tools
+        uses: actions/checkout@v3
+        with:
+          repository: Kong/kong-portal-tools
+          path: kong-portal-tools
+          token: ${{ secrets.GHA_KONG_BOT_READ_TOKEN }}
+
+      - name: copy kong-portal-tools/kong-portal-ci to current dir
```

Comment:
What's the reason to copy files instead of cd in the directory and doing a make build?

Reply:
matter of preference I suppose, I could set `working-directory:` for each make step ::shrug:: this is 1:1 with the previous flow using "bootstrap.mk"

Reply:
which reminds me that file can go, do you have a compelling reason to use  `working-directory:`?

Reply:
`working-directory` seems to be the right call IMO

Reply:
I vote that we maintain the existing setup and get this working. Can always revisit and optimize it later